arrows/klv/klv_0601.h

```diff
@@ -225,6 +225,26 @@ KWIVER_ALGO_KLV_EXPORT
 std::ostream&
 operator<<( std::ostream& os, klv_0601_sensor_fov_name value );
 
+// ----------------------------------------------------------------------------
+/// Source of the navigation positioning information.
+enum KWIVER_ALGO_KLV_EXPORT klv_0601_positioning_method_source
```

Comment:
One more thing I missed the first time through:
```suggestion
enum KWIVER_ALGO_KLV_EXPORT klv_0601_positioning_method_source_bit
```
And edit the enums too (e.g. `...SOURCE_BIT_GPS`), to be consistent with `generic_flag_data_bit`, marking these as bit indices and not simple numeric values.